### ISSUE TITLE
Won't open in Cubase 9.5

### ISSUE THREAD

**Issue by caecos** [bsp2/VeeSeeVSTRack#5]
See https://github.com/VCVRack/Rack/issues/1021 for issue report standards and https://github.com/VCVRack/Rack/issues/730 before posting any code or terminal output.

Operating system(s):
Version of Rack if using official binary, or commit hash and branch if compiling from source:
All hardware relevant to your issue (e.g. graphic card model, audio/MIDI device):

**Comment by caecos** [comment]
Cubase send VeeSeeVSTRack in the black list and refuse to reactive it… Cubase crash when we try...

**Comment by djpeterso23662** [comment]
Cubase 8 on Windows 10 locked up on the initial VST scan.

**Comment by bsp2** [comment]
Could you please download the (64bit) trial version of Reaper and check if it works there ?

Others have mentioned that they are using the plugin in Cubase so this may be a general system incompatibility issue (rather than a Cubase specific one).

The plugin (just like Rack) also requires a fairly decent GPU (with up-to-date drivers).

**Comment by caecos** [comment]
Work perfectly in Reaper 64 bits but still won't work in Cubase 9.5. I'm using a ASUS Rogue laptop with Core i7...lot of RAM, SSD, etc...

**Comment by bsp2** [comment]
Today I installed the Trial version of Cubase Pro 9.5.30.

The plugins (effect + instrument) were working fine except for one major issue: The effect audio input was silent.

I've uploaded a [new release](https://github.com/bsp2/VeeSeeVSTRack/raw/v0.6/dist/veeseevstrack_0_6_1_win64_bin-26Jul2018.7z) that fixes that.

Regarding the blacklist: I've seen this happen with one of my (very simple) test-plugins when I tried to open a debug console, and yes, Cubase crashes when you try to reactivate the plugin.
My workaround was to rename the plugin dll (i.e. name it "plugin.dll__" instead of "plugin.dll"), start Cubase, exit Cubase, then rename the plugin.dll back to its original name.
